components/ShoppingCart.styled.js

```diff
@@ -21,7 +22,98 @@ const CartItemList = styled.ul`
 `;
 
 const TopInfoSection = styled.section`
-  border: solid black;
+  position: relative;
+  border-bottom: solid lightgray;
+  width: 100%;
+  padding: 0px 10px;
```

Comment:
you don't need a unit here

```suggestion
  padding: 0 10px;
```

Reply:
- [x] Fixed